src/components/EducationalInstitutesHeader/index.tsx

```diff
@@ -1,19 +1,29 @@
 import React, { useState } from 'react'
 import { useTranslation } from 'react-i18next'
+import { Roles } from '@allTypes/reduxTypes/usersStateTypes'
 import Filters from '@components/Educationalnstitutes/Filters'
 import { FilterIcon } from '@components/Icons/FilterIcon'
 import { Translation } from '@constants/translations'
+import { useAppSelector } from '@redux/hooks'
+import { usersSelector } from '@redux/slices/users'
 import { Button } from '@uiComponents/Button'
+import { useRouter } from 'next/router'
 
 export const EducationalInstitutesHeader = () => {
   const [t] = useTranslation()
 
   const [active, setActive] = useState(true)
+  const { role } = useAppSelector(usersSelector.user)
+  const router = useRouter()
 
   const openFilter = () => {
     setActive(!active)
   }
 
+  const creatEducationalInstitutes = () => {
+    router.push('/educational-institutes/create-institute')
+  }
+
```

Comment:
function isn't creating institutes, it's just changing route

Reply:
done